cmd/manager/main.go

```diff
@@ -107,8 +108,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	//Create metrics endpoint and register metrics
```

Comment:
I think all this metric logic and calling of the upstream metric package should be contained inside of our local metrics package. This means we do not need to import 	`metrics "github.com/openshift/aws-account-operator/pkg/metrics"`
just to pass metrics into `metricsServer := custommetrics.NewBuilder().WithPort("9090").WithPath("/metrics").`


We can just define a function in `metrics "github.com/openshift/aws-account-operator/pkg/metrics"` and pass in port and path. This will allow all metrics related logic to be self contained.

Reply:
I disagree. The operator-custom-metrics API holds all the logic for starting a prometheus endpoint to expose metrics and can exist and be used completely separately from our definitions of metrics. So combining the two confuses the functions of the two APIs.

Reply:
It makes sense to me that the localmetrics pkg configures and updates the mertics it knows about and registers those with the operator-custom-metrics pkg. This is how I'd expect it to be configured, if we write a function within the localmetrics package to do this for us its likely this implementation would differ from the next operators implementation which we should avoid. We should add this as an example to the operator-custom-metrics README to ensure consistency.